releasenotes/notes/pulse_parameter_manager_compat_with_ParameterVector-7d31395fd4019827.yaml

```
@@ -0,0 +1,10 @@
+---
+prelude: >
+
+features_pulse:
+  - |
+    The `ParameterManager` now supports the `ParameterVector` class. This enables
+    the use of `ParameterVector` in pulse `Schedule` and `ScheduleBlock` objects when using their method
+    `assign_parameters` to assign simultaneously a list of parameter values.
+
+
```

This needs to be formatted as ReStructuredText rather than markdown. Here is I would phrase since it is a user-facing note and users do not generally work with `ParameterManager` directly.

```suggestion
features_pulse:
  - |
    The ``assign_parameters`` methods of :class:`.Schedule` and :class:`.ScheduleBlock`
    now support assigning a :class:`.ParameterVector` to a list of parameter values
    simultaneously in addition to assigning individual :class:`.Parameter` instances to
    individual values.
```